toonz/sources/toonzlib/tstageobjectspline.cpp

```diff
@@ -113,13 +113,31 @@ TStageObjectSpline::TStageObjectSpline()
     , m_id(-1)
     , m_idBase(std::to_string(idBaseCode++))
     , m_name("")
+    , m_active(false)             // Needed for T2D compatibility
+    , m_color(0)                  // Needed for T2D compatibility
+    , m_steps(10)                 // Needed for T2D compatibility
+    , m_width(1)                  // Needed for T2D compatibility
+    , m_externalDataFound(false)  // Needed for T2D compatibility
     , m_isOpened(false) {
   double d = 30;
   std::vector<TThickPoint> points;
   points.push_back(TPointD(0, 0));
   points.push_back(TPointD(d, 0));
   points.push_back(TPointD(2.0 * d, 0));
   m_stroke = new TStroke(points);
+
+  // Needed for T2D compatibility
+  m_interpolationStroke.push_back(TPointD(-40, 0));
+  m_interpolationStroke.push_back(TPointD(-20, 0));
+  m_interpolationStroke.push_back(TPointD(-20, 0));
+  m_interpolationStroke.push_back(TPointD(0, 0));
+  m_interpolationStroke.push_back(TPointD(65, 65));
+
+  m_interpolationStroke.push_back(TPointD(935, 935));
+  m_interpolationStroke.push_back(TPointD(1000, 1000));
+  m_interpolationStroke.push_back(TPointD(1020, 1000));
+  m_interpolationStroke.push_back(TPointD(1020, 1000));
+  m_interpolationStroke.push_back(TPointD(1040, 1000));
```

Comment:
I think setting the default values of `m_interpolationStroke` here is not needed, since the values will always be replaced when loading the T2D file.